src/components/ProgressBar/ProgressBar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div
+    role="progressbar"
+    title="Progress"
+    aria-live="polite"
+    :aria-valuenow="percentage ? percentage : ''"
+    :aria-valuemin="percentage ? 0 : ''"
+    :aria-valuemax="percentage ? 100 : ''"
+    :aria-busy="percentage > 0 && percentage < 100"
+    class="text-left"
+  >
+    <div>
+      Progress
+      <span v-if="percentage"> - {{ percentage }}%</span>
+      <span
+        v-else
+        data-testid="dots"
+      >
+        <span
+          v-for="dot in 3"
+          :key="dot"
+          aria-hidden="true"
+          class="dots animate-ping ml-0.5"
+        >.</span>
+      </span>
+    </div>
+    <div class="mt-2 h-1.5 w-80 bg-gray-100 flex">
+      <span
+        data-testid="innerbar"
+        :style="percentage?`width:${percentage}%`:''"
```

Comment:
nit: I think you don't have the linter for this project, so things look a bit weird

Reply:
ooh what linter/setting is this? you mean there should be some spaces right? I'll fix manually